dll/win32/browseui/shellbrowser.cpp

```diff
@@ -786,11 +786,13 @@ HRESULT CShellBrowser::Initialize()
 
     fToolbarProxy.Initialize(m_hWnd, clientBar);
 
-
     // create status bar
-    fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
-                    SBT_NOBORDERS | SBT_TOOLTIPS, 0, 0, 500, 20, m_hWnd, (HMENU)0xa001,
-                    _AtlBaseModule.GetModuleInstance(), 0);
+    DWORD dwStatusStyle = WS_CHILD | WS_CLIPSIBLINGS | SBARS_SIZEGRIP | SBARS_TOOLTIPS;
+    if (fStatusBarVisible)
+        dwStatusStyle |= WS_VISIBLE;
+    fStatusBar = CreateWindowExW(0, STATUSCLASSNAMEW, NULL, dwStatusStyle,
+                                 0, 0, 500, 20, m_hWnd, (HMENU)FCIDM_TB_UPFOLDER,
+                                 _AtlBaseModule.GetModuleInstance(), 0);
```

Comment:
Btw why CreateWindow got replaced by CreateWindowExW? Arguments look the same except the first one added 0.

Reply:
That was a review request of Hermes. Look upwards.

Reply:
Hermes' suggestion in https://github.com/reactos/reactos/pull/5445#discussion_r1264544598 was to use an in-class call (with `::` preceeding it), here I see just a normal call. I wonder if this changes behavior somehow.

Reply:
As long as no inherited class (nor the current one) implement a CreateWindow(Ex) function, this would be ok, but since we are never sure, and if the expectation here is to call the actual user32 CreateWindow(Ex) function, then one would better use the scope operator on the global namespace. i.e. `::CreateWindowExW` in the actual example above.